includes/forms/functions.php

```diff
@@ -1285,9 +1281,18 @@ function give_admin_form_goal_stats( $form_id ) {
 		( 'donors' === $goal_stats['format'] ? __( 'donors', 'give' ) : ( 'donation' === $goal_stats['format'] ? __( 'donations', 'give' ) : '' ) )
 	);
 
-	if ( $goal_stats['raw_actual'] >= $goal_stats['raw_goal'] ) {
+    /*if (is_archive()){
+        $html .= sprintf( '<span style="opacity: 0" class="give-admin-goal-achieved"><span class="dashicons dashicons-star-filled"></span> %s</span>', __( 'Goal achieved', 'give' ) );
+    }elseif ( $goal_stats['raw_actual'] >= $goal_stats['raw_goal']) {
 		$html .= sprintf( '<span class="give-admin-goal-achieved"><span class="dashicons dashicons-star-filled"></span> %s</span>', __( 'Goal achieved', 'give' ) );
-	}
+	}*/
+
+    $opacity = $goal_stats['raw_actual'] >= $goal_stats['raw_goal'] ? 1 : 0;
+    $html .= sprintf(
+        '<span style="opacity:%s" class="give-admin-goal-achieved"><span class="dashicons dashicons-star-filled"></span> %s</span>',
+        apply_filters('give_admin_goal_progress_achieved_opacity', $opacity),
+        __('Goal achieved', 'give')
+    );
```

Comment:
I'm getting this error on legacy view:

<img width="1318" alt="Screenshot 2024-08-16 at 11 02 55 AM" src="https://github.com/user-attachments/assets/6eaff909-9ba0-4e49-89a3-39da7bb0d78f">

Reply:
Resolved in bd5028b020901394eae3c50561ae3249f284f8ea